Tools/Scripts/import-webdriver-tests

```diff
@@ -103,7 +104,7 @@ importers = []
 
 if options.w3c:
     importer_config_path = webkit_finder.path_from_webkit_base('WebDriverTests', 'imported', 'w3c', 'importer.json')
-    download_path = webkit_finder.path_from_webkit_base('WebKitBuild', 'w3c-tests', 'web-platform-tests')
+    download_path = WPTPaths.default_wpt_checkout_path(webkit_finder)
```

Comment:
cc @lauromoura; the changes both here and `wpt_runner.py` are probably of interest to you, given they're how your ports run the WebDriver tests.